src/kirby/components/icon/custom-icon-settings.ts

```diff
@@ -1,8 +1,15 @@
 import { InjectionToken } from '@angular/core';
 
+export interface CustomIcon {
```

Comment:
Guess it's just an Icon definition now (not really custom anymore, as it's used by Kirby itself)
```suggestion
export interface Icon {
```